consul/tools/chocolateyInstall.ps1

```diff
@@ -1,6 +1,7 @@
 try {
   $binariesPath = $(Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) "..\binaries\")
   $toolsPath = (Split-Path -Parent $MyInvocation.MyCommand.Definition)
+  $shimsPath = $(Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) "..\..\..\bin\")
```

Comment:
This assumes that the bin directory is three subdirectories down from the choco package directory, but this might not hold true for all cases.